### PR TITLE
Cookie move tests

### LINKED ISSUE #1178: Relocate `tests` folder to top project level
### Problem

Currently, the tests are located under `src/regolith/tests`.

### Proposed solution

move `tests` at the top directly level, side by side with `src`

### PR DESCRIPTION
**Pull request by tinatn29**
Fix #1178 
- move tests from src/regolith/tests to ./tests (sorry I used `mv` and not `git mv`. I tried `git mv` but it didn't work with some __pycache__ complaints...) 
- make `flake8` in `.pre-commit-config.yaml` exclude tests/test_helpers.py and tests/test_commands.py
- fix minor typos (codespell)

## Results:
- pytest is passing locally (with 34 skipped and 57 warnings, same as before the move)
- `pre-commit run --all-files` now passes

### CODE REVIEW

File: tests/conftest.py
Comment (tinatn29) [inline tests/conftest.py:1]: I copied over everything from the old `src/regolith/tests/conftest.py`

File: .pre-commit-config.yaml
Comment (tinatn29) [inline .pre-commit-config.yaml:1]: ignoring test_helpers.py (flake8: lines too long) and test_commands.py so I can skip conftest import

**Comment by bobleesj** [comment]
@tinatn29 Hi Tina, 

could you try to re-run CI here now using `git commit --allow-empty -m "Re-cun CI with empty commit`? Hopefully it passes now.

**Comment by sbillinge** [comment]
@tinatn29 if you can make a push to this PR it should retrigger the CI.  Bob showed above how to make an emtpy commit to do that.  Or if there is any edit to make, you can do that too.

**Comment by tinatn29** [comment]
just did!

**Pull request merged**